bundles/org.openhab.binding.solarlog/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -29,12 +29,12 @@
 		<config-description>
 			<parameter name="url" type="text" required="true">
 				<label>URL</label>
-				<description>URL of the Solar-Log web interface (e.g. http://solar-log)</description>
+				<description>URL of the Solar-Log web interface (e.g., http://solar-log)</description>
 				<default>http://solar-log</default>
 			</parameter>
 			<parameter name="refreshInterval" type="integer" min="15">
 				<label>Refresh Interval</label>
-				<description>States how often a refresh shall occur (in s).</description>
+				<description>How often to refresh (in seconds).</description>
```

Comment:
Can you also update the I18N properties file?

https://github.com/openhab/openhab-addons/blob/25b5dcb554e8aaa27dc61fb4d5036e09189e84a8/bundles/org.openhab.binding.solarlog/src/main/resources/OH-INF/i18n/solarlog.properties#L14